py/desisim/scripts/quickgen.py

```diff
@@ -412,10 +508,21 @@ def main(args=None):
             nivar[j, i, :num_pixels] = 1.0 / output['variance_electrons']
 
             # Get results for our flux-calibrated output file.
-            assert output['observed_flux'].unit == u.erg / (u.cm**2 * u.s * u.Angstrom)
             cframe_observedflux[j, i, :num_pixels] = 1e17 * output['observed_flux']
             cframe_ivar[j, i, :num_pixels] = 1e-34 * output['flux_inverse_variance']
 
+            # Fill brick arrays from the results.
+            if i==0:
+                camera = 'b'
+            elif i==1:
+                camera = 'r'
+            else:
+                camera = 'z'
```

Comment:
I think the qsim.camera_output results are not guaranteed to be in the order b, r, z.  There must be a better way to derive their name.